doc/source/user_guide/io.rst

```diff
@@ -332,6 +332,11 @@ encoding : str, default ``None``
   Encoding to use for UTF when reading/writing (e.g. ``'utf-8'``). `List of
   Python standard encodings
   <https://docs.python.org/3/library/codecs.html#standard-encodings>`_.
+encoding_errors : str, default 'strict'
```

Comment:
Sorry, I think I led you astray here. This should be removed, since this section is for `read_csv`, and the change is just for `to_csv`.